internal/bundles/manifest.go

```diff
@@ -202,6 +203,14 @@ func (manifest *Manifest) AddFile(path string, fileMD5 []byte) {
 	manifest.Files[path] = ManifestFile{
 		Checksum: hex.EncodeToString(fileMD5),
 	}
+
+	// Update manifest content category if a file being added is a site configuration
+	for _, ymlFile := range util.KnownSiteYmlConfigFiles {
+		if strings.HasSuffix(path, ymlFile) {
```

Comment:
I believe there is a potential problem here. While this will match what you intend, such as `/usr/bill/project/demo/_site.yml`, it would also match what I believe you don't want in that `/usr/bill/project/demo/abc123_mystuff_site.yml`, since the search is simply using `HasSuffix` rather than getting the filename from the path and then comparing that file name with the known site YML Config Files.

Perhaps you could consider using `internal/util/fspath.go`, and comparing the Base() against the known filenames? i'm not sure if you'll need to make that a lowercase test or not.

Or perhaps I'm missing something here?

Reply:
Adjustment done, got rid of the `HasSuffix` method, we can compare this filename directly since the possible site configuration files we look for must be present at the project root, we can ignore multi step paths. Thanks for noticing this!